### ISSUE TITLE
New GVL location

### ISSUE THREAD

**Issue by patmmccann** [prebid/prebid-server#2697]
https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework/pull/334/files indicates a new location for the EU GVL beginning "mid-May". The CA GVL, with shared vendor id's but a different list of valid vendors, will remain at https://vendor-list.consensu.org/v2/vendor-list.json (temporarily?) 

Reference:
 https://github.com/prebid/prebid-server/blob/9dc2d5fb36215197e01bb3e12227dd0f09eb5b7c/gdpr/vendorlist-fetching_test.go#L145

**Comment by bretg** [comment]
Here are the details:

1. PBS must parse the `tcfPolicyVersion` field out of the incoming TCF string in addition to `vendorListVersion`
2. Set gvlSpecificationVersion.
    a. If tcfPolicyVersion<4, set gvlSpecificationVersion=2
    b. If tcfPolicyVersion==4, set gvlSpecificationVersion=3
    c. otherwise, the incoming TCF string is invalid. Blank it out (no consent), emit an error in debug output, and log an error to the error log at N% sampling. The error should contain the value of the received tcfPolicyVersion.
3. Resolve the URL for the GVL list: 

```
https://vendor-list.consensu.org/v##gvlSpecificationVersion##/archives/vendor-list-v##vendorListVersion##.json 
```

**Comment by bretg** [comment]
Discussed in committee
- timeline is that we need to support the v3 path in the next couple of months
- the v2 path may go away in Q3/Q4
- we need to know if there are GVL JSON changes that we have to accommodate
 
Waiting for the PR to get merged before flagging as ready-for-dev.

**Comment by bretg** [comment]
Here's an analysis of the impact on the GVL v3 format changes to Prebid Server. In short, the JSON schema will need to be updated, but no TCF math changes. 

- added dataDeclaration array. "An array of positive integers that represent the data categories declared by the vendor". This seems to be related to a requirement that vendors need a "List of Categories of data collected across Purposes". (Unlikely something that Prebid Server can take action on.)
- added dataRetention object. "an object that contains the data retention for the purpose and specialPurpose declared by the vendor" (no PBS changes)
- added urls array. "An array of url objects representing language, policy url and legitimate interest url" (no PBS changes)
- Removal of the policyUrl field. (no PBS changes, it doesn't consume this field)
- certain values are no longer allowed in legIntPurposes (no PBS changes)

**Comment by SyntaxNode** [comment]
@bretg TCF 2.2 / GVL 3 includes a new purpose. Purpose 11 is defined as "Use limited data to select content". Does PBS need to take action on this new purpose?

**Comment by bretg** [comment]
Released with PBS-Go 0.260

**Comment by bretg** [comment]
Done with PBS-Java 1.123
